api/src/pcapi/routes/public/individual_offers/v1/products.py

```diff
@@ -313,29 +313,29 @@ def get_product(product_id: int) -> serialization.ProductOfferResponse:
     return serialization.ProductOfferResponse.build_product_offer(offer)
 
 
-@blueprint.v1_blueprint.route("/products/ean/<string:ean>", methods=["GET"])
+@blueprint.v1_blueprint.route("/products/ean", methods=["GET"])
 @spectree_serialize(
     api=blueprint.v1_product_schema,
     tags=[constants.PRODUCT_EAN_OFFER_TAG],
-    response_model=serialization.ProductOfferResponse,
+    response_model=serialization.ProductOffersByEanResponse,
 )
 @api_key_required
-def get_product_by_ean(ean: str) -> serialization.ProductOfferResponse:
+def get_product_by_ean(query: serialization.GetProductsListByEansQuery) -> serialization.ProductOffersByEanResponse:
     """
     Get a product offer using its European Article Number (EAN-13).
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** Le commentaire n'est plus exact